src/main/java/pl/betoncraft/betonquest/compatibility/jobsreborn/Event_AddLevel.java

```diff
@@ -35,36 +36,43 @@
 
     public Event_AddLevel(Instruction instruction) throws InstructionParseException {
         super(instruction, true);
+
         if (instruction.size() < 3) {
             throw new InstructionParseException("Not enough arguments");
         }
+
         for (Job job : Jobs.getJobs()) {
             if (job.getName().equalsIgnoreCase(instruction.getPart(1))) {
                 sJobName = job.getName();
+
                 try {
                     this.nAddLevel = Integer.parseInt(instruction.getPart(2));
                 } catch (Exception e) {
                     throw new InstructionParseException("NUJobs_AddLevel: Unable to parse the level amount", e);
                 }
+
                 return;
             }
         }
+
         throw new InstructionParseException("Jobs Reborn job " + instruction.getPart(1) + " does not exist");
     }
 
     @Override
     protected Void execute(String playerID) {
         Player oPlayer = PlayerConverter.getPlayer(playerID);
+		JobsPlayer jPlayer = Jobs.getPlayerManager().getJobsPlayer(oPlayer);
+		if (jPlayer == null)
+			return null;
 
-        List<JobProgression> oJobs = Jobs.getPlayerManager().getJobsPlayer(oPlayer).getJobProgression();
+        List<JobProgression> oJobs = jPlayer.getJobProgression();
         for (JobProgression oJob : oJobs) {
             if (oJob.getJob().getName().equalsIgnoreCase(sJobName)) {
-                //User has the job, return true
                 oJob.setLevel(this.nAddLevel + oJob.getLevel());
-                if (oJob.getLevel() > oJob.getJob().getMaxLevel())
```

Comment:
is this not needed anymore?

Reply:
Np, because `oJob.getJob().getMaxLevel();` its doing nothing.